src/redux/index.js

```diff
@@ -44,5 +44,5 @@ export const store = createStore(
 );
 
 export const persistor = persistStore(store, {}, () => {
-  store.dispatch(uploadActions.refreshIncompleteUploads());
+  // store.dispatch(uploadActions.refreshIncompleteUploads());
```

Comment:
@EdmundMai what was this needed for? Turning it on fails the e2e tests for streams.

Reply:
i think we can remove this, i don't think it's being used

Reply:
Ok, i'll delete this in a separate PR. this one is getting too big.